packages/cozy-stack-client/src/DocumentCollection.spec.js

```diff
@@ -176,7 +176,7 @@ describe('DocumentCollection', () => {
       await collection.all({ keys: ['abc', 'def'] })
       expect(client.fetchJSON).toHaveBeenCalledWith(
         'GET',
-        '/data/io.cozy.todos/_all_docs?include_docs=true&keys=[%22abc%22,%22def%22]'
+        '/data/io.cozy.todos/_all_docs?include_docs=true&limit=100&keys=[%22abc%22,%22def%22]'
```

Comment:
This commit should be squashed/rebased, we should not have any commit that is red otherwise we can have bad surprises when bisecting.